components/servo/tests/common/mod.rs

```diff
@@ -88,7 +88,7 @@ pub(crate) struct WebViewDelegateImpl {
     pub(crate) number_of_controls_hidden: Cell<usize>,
 }
 
-#[allow(dead_code)]
+#[expect(dead_code)]
```

Comment:
These are used in some of the test suites, but not the others: https://github.com/servo/servo/commit/9c86ef449dc6ccb9bdf74ff6bbaffe3b27030276

I think if you run `./mach test-unit -d` now, they will no longer compile. Instead, please add a comment why this should be allowed